kwriteconfig5: add page (#5292)
tldr-pages/tldr: pages/linux/kwriteconfig5.md

# kwriteconfig5

> Write KConfig entries for KDE Plasma.
> More information: <https://userbase.kde.org/KDE_System_Administration/Configuration_Files>.

> - Set a value for a global configuration:

Reviewer: This can be simplified:
```suggestion
- Set a global configuration key:
```

`kwriteconfig5 --group {{group_name}} --key {{key_name}} {{value}}`

> - Set a value in a specific configuration file:

Reviewer: ```suggestion
- Set a key in a specific configuration file:
```

`kwriteconfig5 --file {{path/to/file}} --group {{group_name}} --key {{key_name}} {{value}}`

> - Delete the value from a key:

Reviewer: ```suggestion
- Delete a key:
```

`kwriteconfig5 --group {{group_name}} --key {{key_name}} --delete`

> - Use Systemd to start the Plasma session when available:

Reviewer: ```suggestion
- Use systemd to start the Plasma session when available:
```

https://www.freedesktop.org/wiki/Software/systemd/

>Yes, it is written **systemd**, not **system D** or **System D**, or even **SystemD**.

> `kwriteconfig5 --file {{startkderc}} --group {[General]} --key {{systemdBoot}} {{true}}`

Reviewer: Mistake?
```suggestion
`kwriteconfig5 --file {{startkderc}} --group {{General}} --key {{systemdBoot}} {{true}}`
```

Author: Yep.

> - Hide title bars when maximized (like Ubuntu):

Reviewer: ```suggestion
- Hide the title bar when a window is maximized (like Ubuntu):
```

`kwriteconfig5 --file {{~/.config/kwinrc}} --group {{Windows}} --key {{BorderlessMaximizedWindows}} {{true}}`

> - Open KRunner with the Meta (Windows) key:

Reviewer: Meta is also Command on macOS.
```suggestion
- Open KRunner with the Meta (Command/Windows) key:
```

# kwriteconfig5

> Write KConfig entries for KDE Plasma.
> More information: <https://userbase.kde.org/KDE_System_Administration/Configuration_Files>.

- Display help:

`kwriteconfig5 --help`

- Set a global configuration key:

> `kwriteconfig5 --group {{group_name}} --key {{key_name}} {{value}}`

Reviewer: ```suggestion
`kwriteconfig5 --group {{group_name}} --key {{key}} {{value}}`
```

- Set a key in a specific configuration file:

> `kwriteconfig5 --file {{path/to/file}} --group {{group_name}} --key {{key_name}} {{value}}`

Reviewer: ```suggestion
`kwriteconfig5 --file {{path/to/file}} --group {{group_name}} --key {{key}} {{value}}`
```

- Delete a key:

> `kwriteconfig5 --group {{group_name}} --key {{key_name}} --delete`

Reviewer: ```suggestion
`kwriteconfig5 --group {{group_name}} --key {{key}} --delete`
```

- Use systemd to start the Plasma session when available:

`kwriteconfig5 --file {{startkderc}} --group {{General}} --key {{systemdBoot}} {{true}}`

- Hide the title bar when a window is maximized (like Ubuntu):

`kwriteconfig5 --file {{~/.config/kwinrc}} --group {{Windows}} --key {{BorderlessMaximizedWindows}} {{true}}`

> - Open KRunner with the Meta (Command/Windows) key:

Reviewer: What about this?
```suggestion
- Configure KRunner to open with the Meta (Command/Windows) global hotkey:
```

Author: Yep exactly.

Author: :thinking: As a user you're configuring the Plasma Shell to open KRunner with the Meta key.
